cmd/kubeadm/app/util/apiclient/dryrun.go

```diff
@@ -607,7 +607,6 @@ clusters:
 contexts: null
 current-context: ""
 kind: Config
-preferences: {}
```

Comment:
this broke one of kubeadm's e2e tests for dryrun which under the hood utilizes a fake client with reactors and fixtures. more particularly it dryruns the bootstrap token and cluster-info configmap validation.

we must also update line 614 with the expected JWS signature after the contents of the cluster-info have changed.

- https://github.com/kubernetes/kubeadm/issues/3202

fix is pending, looking for lgtm:
- https://github.com/kubernetes/kubernetes/pull/131880

Reply:
Sorry about that, didn't catch that bit anywhere.